.github/workflows/build_and_release.yml

```diff
@@ -396,32 +403,31 @@ jobs:
             aubio fftw glm nlohmann-json fmt
            brew link --force --overwrite ffmpeg@4
 
+      - name: Setup Python
+        id: setup_python
+        uses: actions/setup-python@v4 
+        with:
+            python-version: 'pypy3.9' 
```

Comment:
if we're using python for the first time, why lock on a python version over 2 years old?

Reply:
It's the one that comes bundled with the latest xcode command line tools and, do we honestly need a newer one?